### PR TITLE
AppRegistry integration

### PR DESCRIPTION
**Pull request by sandimciin**
*Issue #, if available:*

*Description of changes:*
Added resources for AppRegistry nested stack configuration.
Updated requests-aws4auth package to v1.2.2.
Updated s3 url for Media Insights template.
Changed parameter values from "true" to "Yes" for github workflows

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.1.0] - 03/20/2023
+
+### New:
```
Comment (dch90) [inline CHANGELOG.md:9]: When we were working on the Media Insights release, the `viperlight - publisher check` barked at us for not following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) to the t.

i.e. adding `:` after each `New`, `Changed`, etc

File: deployment/content-localization-on-aws.yaml [hunk 2]
```diff
@@ -44,7 +50,7 @@ Resources:
     Properties:
       TemplateURL: !Join
         - ""
-        - - "https://solutions-reference.s3.amazonaws.com/aws-media-insights-engine/"
+        - - "https://solutions-reference.s3.amazonaws.com/media-insights-on-aws/"
```
Comment (dch90) [inline deployment/content-localization-on-aws.yaml:53]: The template name for the new release got changed.
It used to be `media-insights-stack.template`
Now it is `media-insights-on-aws-stack.template`

check line 58

File: CHANGELOG.md (hunk 1 above)
Comment (colinazn) [reply CHANGELOG.md]: Yep, the post-scan stage in the pipeline scans this file and has a lot to say about it.

**Review by colinazn — COMMENTED** [review]
I think the PR workflow is failing when we try to create a S3 bucket because of the invalid credentials and the build-mie-dev-us-west-2 is failing because of a missing region configuration. Also, you'll probably need to change the bucket names so the build scripts don't try to upload the build artifacts.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,51 +4,62 @@ All notable changes to this project will be documented in this file.
...
+
 ## [2.0.2] - 01/11/2023
 
-### Fixed:
+### Fixed
 
-* Updated python packages
-* Replaced deprecated python command with python3
+- Updated python packages
+- Replaced deprecated python command with python3
```
[the remark this answers is not in the document]
Comment (sandimciin) [reply CHANGELOG.md:23]: does viperlight care about this syntax?

**Comment by sandimciin** [comment]
redo PR

**Pull request closed without merging**